packages-exp/functions-compat/src/index.ts

```diff
@@ -18,19 +18,33 @@
 import firebase from '@firebase/app-compat';
 import { name, version } from '../package.json';
 import { registerFunctions } from './register';
-import * as types from '@firebase/functions-types';
+import { FirebaseFunctions as FunctionsCompat } from '@firebase/functions-types';
+import { Functions as FunctionsExp } from '@firebase/functions-exp';
 
+declare module '@firebase/functions-exp' {
+  export function httpsCallable<RequestData = unknown, ResponseData = unknown>(
+    functionsInstance: FunctionsCompat | FunctionsExp,
```

Comment:
```suggestion
    functionsInstance: FunctionsCompat
```
We can just define an overload that only accepts the compat instance

Reply:
And similarly for other places.

Reply:
Done.